crates/autopilot/src/database/orders.rs

```diff
@@ -20,12 +21,25 @@ impl Postgres {
             .with_label_values(&["orders_for_tx"])
             .start_timer();
 
-        database::orders::order_executions_in_tx(ex, &ByteArray(tx_hash.0), auction_id)
-            .map(|result| match result {
-                Ok(execution) => execution.try_into().map_err(Into::into),
-                Err(err) => Err(anyhow::Error::from(err)),
-            })
-            .try_collect()
-            .await
+        let mut order_executions = Vec::new();
+
+        let executions =
```

Comment:
nanonit

```suggestion
        let db_executions =
```

to make it clear these are different types.

Reply:
Updated.